src/helpers/get-initial-active-slide-index.ts

```diff
@@ -0,0 +1,12 @@
+function getInitialActiveSlideIndex(
+  index: number | null,
+  targetId: string | null | undefined,
+): number {
+  if (index !== null) {
+    return index
+  }
+
+  return targetId ? parseInt(targetId, 10) - 1 : 0
```

Comment:
`targetId` is a string, so it can be something like `first-pic` and there NaN will be produced 🥲

but this logic was in `gallery.ts` before this PR

Reply:
Maybe fix it in this PR, or create separate issue?

Reply:
I prefer separate issue - https://github.com/dromru/react-photoswipe-gallery/issues/1050